Automattic/jetpack
# Admin bar: clean up WPCOM_ADMIN_BAR_UNIFICATION feature flag

Related to: https://github.com/Automattic/wp-calypso/issues/95335

## Proposed changes:

We introduced the `WPCOM_ADMIN_BAR_UNIFICATION` flag to implement pfsHM7-1kz-p2. Now that it's fully rolled out, it's time to clean this up.

This PR effectively removes:

- the `Masterbar` class which provided the custom wpcom admin bar
- the custom CSS overrides for Core color schemes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Test all the following combination of:
   - Simple vs Atomic
   - Default vs Classic view
- For each case, check for wp-admin and Calypso screens; verify that nothing breaks in the sidebar and admin bar.
